File: igtampe.cdbfs.frontend/src/model/CdbfsDrive.ts
import Editable from "./Editable"

export default class CdbfsDrive extends Editable {
    public constructor(
        public id: number,
        public name: string,
        public fileCount: number,
        public folderCount: number,
        public size: number,
        createTs: string, createUserId: string, updateTs: string, updateUserId: string,
    ) { super(createTs, createUserId, updateTs, updateUserId) }
}

File: Igtampe.CDBFS.Common/CdbfsDrive.cs
namespace Igtampe.CDBFS.Common {
    public class CdbfsDrive : Editable {
        public int Id { get; set; } = 0;
        public string Name { get; set; } = "";
        public int FileCount { get; set; } = 0;
        public int FolderCount { get; set; } = 0;
        public long? Size { get; set; } = 0;
    }
}
